runtime/vm/jvminit.c

```diff
@@ -2902,59 +2916,59 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 
 		/* Since these two options are -XX, we don't need to consume them as they will be ignored */
 
-		if (strcmp(testString, VMOPT_XXNOSTACKTRACEINTHROWABLE) == 0) {
+		if (FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXNOSTACKTRACEINTHROWABLE, NULL) != -1) {
```

Comment:
@pshipton The fix was made here where `XX` options are consumed here (where previously, they were not)

Reply:
ok, sorry. I didn't realize your earlier comment meant you had updated the code in this PR.